langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/internal/client/DefaultMistralAiClient.java

```diff
@@ -132,11 +129,11 @@ public void onEvent(EventSource eventSource, String id, String type, String data
                     handler.onComplete(response);
                 } else {
                     try {
-                        MistralAiChatCompletionResponse chatCompletionResponse = GSON.fromJson(data, MistralAiChatCompletionResponse.class);
+                        MistralAiChatCompletionResponse chatCompletionResponse = OBJECT_MAPPER.readValue(data, MistralAiChatCompletionResponse.class);
                         MistralAiChatCompletionChoice choice = chatCompletionResponse.getChoices().get(0);
 
                         String chunk = choice.getDelta().getContent();
-                        if (isNotNullOrBlank(chunk)) {
+                        if (isNotNullOrEmpty(chunk)) {
```

Comment:
worth extracting this bugfix into a separate PR?

Reply:
uhm maybe for tracking purpose. It's a trivial change but I can fix it in a separate PR for sure.